prusti-interface/src/specs/mod.rs

```diff
@@ -7,6 +7,7 @@ use crate::{
     PrustiError,
 };
 use log::debug;
+use prusti_common::config::opt_in_verification;
```

Comment:
For consistency; we usually just import `config`.
```suggestion
use prusti_common::config;
```